openeo/NDVI_Timeseries.ipynb

```diff
@@ -0,0 +1,427 @@
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th>polygon</th>\n",
+       "      <th>0</th>\n",
+       "      <th>1</th>\n",
+       "      <th>2</th>\n",
+       "      <th>3</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>date</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>2020-06-01 00:00:00+00:00</th>\n",
+       "      <td>0.748476</td>\n",
+       "      <td>0.495063</td>\n",
+       "      <td>0.655510</td>\n",
+       "      <td>0.486538</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2020-06-04 00:00:00+00:00</th>\n",
+       "      <td>0.028493</td>\n",
+       "      <td>0.026855</td>\n",
+       "      <td>0.026339</td>\n",
+       "      <td>0.026769</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2020-06-06 00:00:00+00:00</th>\n",
+       "      <td>0.760013</td>\n",
+       "      <td>0.501150</td>\n",
+       "      <td>0.694817</td>\n",
+       "      <td>0.394088</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2020-06-09 00:00:00+00:00</th>\n",
+       "      <td>0.070291</td>\n",
+       "      <td>0.073217</td>\n",
+       "      <td>0.062952</td>\n",
+       "      <td>0.047764</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2020-06-11 00:00:00+00:00</th>\n",
+       "      <td>0.024885</td>\n",
+       "      <td>0.026817</td>\n",
+       "      <td>0.028701</td>\n",
+       "      <td>0.026536</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "polygon                           0         1         2         3\n",
+       "date                                                             \n",
+       "2020-06-01 00:00:00+00:00  0.748476  0.495063  0.655510  0.486538\n",
+       "2020-06-04 00:00:00+00:00  0.028493  0.026855  0.026339  0.026769\n",
+       "2020-06-06 00:00:00+00:00  0.760013  0.501150  0.694817  0.394088\n",
+       "2020-06-09 00:00:00+00:00  0.070291  0.073217  0.062952  0.047764\n",
+       "2020-06-11 00:00:00+00:00  0.024885  0.026817  0.028701  0.026536"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "with open(\"output/timeseries.json\") as f:\n",
+    "    ts = timeseries_json_to_pandas(json.load(f)).dropna()\n",
+    "ts.index = pd.to_datetime(ts.index)\n",
+    "ts.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# defining the plotting function\n",
+    "\n",
+    "import matplotlib.pyplot as plt \n",
```

Comment:
you import matplotlib here for a third time :)

Reply:
corrected